npm-shrinkwrap.json

```diff
@@ -4,5929 +4,5487 @@
   "dependencies": {
```

Comment:
Nope not at all, that would defeat the whole purpose of npm-shrinkwrap? npm-shrinkwrap (and at some point we really should switch to yarn as well tbh so it's all automated, it's just an overall superior tool with the same API) makes sure everyone installs the same packages, it wouldn't make any sense to put it in our build process tbh.